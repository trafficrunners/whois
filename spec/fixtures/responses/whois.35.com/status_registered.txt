
Domain Name: zwad.net
Registry Domain ID:
Registrar WHOIS Server: whois.35.com
Registrar URL: http://www.35.com
Updated Date: 2014-08-21 23:48:59
Creation Date: 2010-09-10
Registrar Registration Expiration Date: 2015-09-10
Registrar: 35 Technology Co., Ltd.
Registrar IANA ID: 1316
Registrar Abuse Contact Email: abuse@35.cn
Registrar Abuse Contact Phone: +86.5925391800
Reseller: �������Ϻ��������Ǵ�ƽ�����ƹ�����                          
Domain Status: clientTransferProhibited
Registry Registrant ID:
Registrant Name:baolei
Registrant Organization:jiaxing zhongwang ad co.,ltd.
Registrant Street:126# jiyang rd.
Registrant City:jiaxing
Registrant State/Province:zhejiang
Registrant Postal Code:314000
Registrant Country:CN
Registrant Phone:+86.57382033533
Registrant Phone Ext:
Registrant Fax:+86.57382033533
Registrant Fax Ext:
Registrant Email:aaa@zjcf.com
Registry Admin ID:
Admin Name:baolei
Admin Organization:jiaxing zhongwang ad co.,ltd.
Admin Street:126# jiyang rd.
Admin City:jiaxing
Admin State/Province:zhejiang
Admin Postal Code:314000
Admin Country:CN
Admin Phone:+86.57382033533
Admin Phone Ext:
Admin Fax:+86.57382033533
Admin Fax Ext:
Admin Email:aaa@zjcf.com
Registry Tech ID:
Tech Name:baolei
Tech Organization:jiaxing zhongwang ad co.,ltd.
Tech Street:126# jiyang rd.
Tech City:jiaxing
Tech State/Province:zhejiang
Tech Postal Code:314000
Tech Country:CN
Tech Phone:+86.57382033533
Tech Phone Ext:
Tech Fax:+86.57382033533
Tech Fax Ext:
Tech Email:aaa@zjcf.com
Name Server: ns3.dns-diy.com
Name Server: ns4.dns-diy.com
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System:http://wdprs.internic.net/
>>> Last update of WHOIS database: 2014-08-21 23:48:59 <<<
