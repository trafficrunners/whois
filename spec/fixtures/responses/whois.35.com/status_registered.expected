#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2010-09-10}

#updated_on
  %s %CLASS{time}
  %s %TIME{2014-08-21 23:48:59}

#expires_on
  %s %CLASS{time}
  %s %TIME{2015-09-10}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1316"
  %s.name         == "35 Technology Co., Ltd."
  %s.organization == "35 Technology Co., Ltd."
  %s.url          == "http://www.35.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "baolei"
  %s[0].organization  == "jiaxing zhongwang ad co.,ltd."
  %s[0].address       == "126# jiyang rd."
  %s[0].city          == "jiaxing"
  %s[0].zip           == "314000"
  %s[0].state         == "zhejiang"
  %s[0].country       == nil
  %s[0].country_code  == "CN"
  %s[0].phone         == "+86.57382033533"
  %s[0].fax           == "+86.57382033533"
  %s[0].email         == "aaa@zjcf.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == nil
  %s[0].name          == "baolei"
  %s[0].organization  == "jiaxing zhongwang ad co.,ltd."
  %s[0].address       == "126# jiyang rd."
  %s[0].city          == "jiaxing"
  %s[0].zip           == "314000"
  %s[0].state         == "zhejiang"
  %s[0].country       == nil
  %s[0].country_code  == "CN"
  %s[0].phone         == "+86.57382033533"
  %s[0].fax           == "+86.57382033533"
  %s[0].email         == "aaa@zjcf.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "baolei"
  %s[0].organization  == "jiaxing zhongwang ad co.,ltd."
  %s[0].address       == "126# jiyang rd."
  %s[0].city          == "jiaxing"
  %s[0].zip           == "314000"
  %s[0].state         == "zhejiang"
  %s[0].country       == nil
  %s[0].country_code  == "CN"
  %s[0].phone         == "+86.57382033533"
  %s[0].fax           == "+86.57382033533"
  %s[0].email         == "aaa@zjcf.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns3.dns-diy.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns4.dns-diy.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil

